### PR TITLE
Standardize the Gruntfile.js and gulpfile.js files

### PR DESCRIPTION
**Pull request by scottaddie**
Standardized the quotes and JS strict mode usage throughout all of the Gruntfile.js and gulpfile.js files.

### CODE REVIEW

**Comment by sayedihashimi** [comment]
@scottaddie it looks like the build is failing https://travis-ci.org/OmniSharp/generator-aspnet/builds/86955830.

**Comment by scottaddie** [comment]
@sayedihashimi Sorry about that. I forgot to check in the modified test.

File: templates/Gruntfile.js [hunk 1]
```diff
@@ -2,8 +2,9 @@
   This file in the main entry point for defining grunt tasks and using grunt plugins.
   Click here to learn more. http://go.microsoft.com/fwlink/?LinkID=513275&clcid=0x409
 */
-'use strict';
 module.exports = function(grunt) {
+  "use strict";
```
Comment (peterblazejewicz) [inline templates/Gruntfile.js]: Why moved here?
https://github.com/gruntjs/grunt/blob/master/Gruntfile.js#L10

**Comment by peterblazejewicz** [comment]
@scottaddie 
We can move "use strict" as in modules to standarize:
https://github.com/gruntjs/grunt/blob/master/Gruntfile.js#L10
Otherwise LGTM
Can you rebase squashing commits?

**Comment by scottaddie** [comment]
@peterblazejewicz Will open a new PR with the single commit, as it will be easier for me to figure out :-)

**Comment by peterblazejewicz** [comment]
Sorry for the hassle :)

**Pull request closed without merging**